ionic/githubIonic/src/pages/login/login.ts

```diff
@@ -26,7 +27,8 @@ export class LoginPage {
         this.userService.signIn(this.username, this.password).subscribe(response => {
             localStorage.setItem('user-token', response.access_token);
             localStorage.setItem('refresh-token', response.refresh_token);
-            this.viewCtrl.dismiss();
+             let modal = this.modalCtrl.create(MyApp);
+              modal.present();
```

Comment:
there are other ways, this overlays the app on top of itself. It's a waste of space